features/flows/classification/engine/api/src/main/java/org/opennms/netmgt/flows/classification/error/Errors.java

```diff
@@ -54,6 +54,8 @@
     ErrorTemplate RULE_EXPORTER_FILTER_INVALID = new ErrorTemplate("rule.filter.invalid", "The provided filter ''{0}'' is invalid: {1}");
 
     ErrorTemplate RULE_IP_ADDRESS_INVALID = new ErrorTemplate("rule.ipaddress.invalid", "The provided IP Address ''{0}'' is not valid.");
+    ErrorTemplate RULE_IP_ADDRESS_RANGE_INVALID = new ErrorTemplate("rule.ipaddress.range.invalid", "The provided IP Address range ''{0}'' is not valid.");
```

Comment:
I wonder if we should give here an example of how a valid range should look like?